modules/nativoBidAdapter.js

```diff
@@ -375,3 +466,14 @@ function getLargestSize(sizes, method = area) {
  * @returns The calculated area
  */
 const area = (size) => size[0] * size[1]
+
+/**
+ * Save any filter data from winning bid requests for subsequent requests
+ * @param {Array} filter - The filter data bucket currently stored
+ * @param {Array} filterData - The filter data to add
+ */
+function appendFilterData(filter, filterData) {
+  if (filterData && Array.isArray(filterData) && filterData.length > 0) {
```

Comment:
0 is falsey so you can simply say 

`if (filterData && Array.isArray(filterData) && filterData.length) {`